backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/Setup.kt

```diff
@@ -1,11 +1,413 @@
+    ),
+    listOf(
+        "Stadt",
+        "Schweinfurt",
+        "09662",
+        "https://www.schweinfurt.de/leben-freizeit/ehrenamt-projekte/4362.Bayerische-Ehrenamtskarte.html"
+    ),
+    listOf(
+        "Landkreis",
+        "Schweinfurt",
+        "09678",
+        "https://www.landkreis-schweinfurt.de/service-infos/serviceleistungen-informationen/ServiceInfos/detail/bayerische-ehrenamtskarte-beantragung-903/"
+    ),
+    listOf("Landkreis", "Starnberg", "09188", "http://www.lk-starnberg.de/ehrenamtskarte"),
+    listOf("Stadt", "Straubing", "09263", "http://www.straubing.de/de/buerger-und-soziales/ehrenamt/"),
+    listOf(
+        "Landkreis",
+        "Straubing-Bogen",
+        "09278",
+        "http://www.landkreis-straubing-bogen.de/kultur-bildung-sport-soziales/ehrenamt/"
+    ),
+    listOf(
+        "Landkreis",
+        "Tirschenreuth",
+        "09377",
+        "http://www.kreis-tir.de/verwaltung-organisation/fachbereiche/soziales-ehrenamt/ehrenamtskarte/ehrenamtskarte-bayern/"
+    ),
+    listOf("Landkreis", "Traunstein", "09189", "https://www.traunstein.com/buerger-verwaltung/freiwilligenagentur"),
+    listOf("Landkreis", "Unterallgäu", "09778", "http://www.unterallgaeu.de/ehrenamtskarte"),
+    listOf(
+        "Stadt",
+        "Würzburg",
+        "09663",
+        "https://www.wuerzburg.de/themen/gesundheit-soziales/aktivbuero/ehrenamt/417110.Bayerische-Ehrenamtskarte.html"
+    ),
+    listOf(
+        "Landkreis",
+        "Würzburg",
+        "09679",
+        "http://www.landkreis-wuerzburg.de/Politik_Behörde/Servicestelle_Ehrenamt/Bayerische_Ehrenamtskarte/"
+    ),
+    listOf("Landkreis", "Weißenburg-Gunzenhausen", "09577", "http://www.altmuehlfranken.de/ehrenamtskarte"),
+    listOf("Stadt", "Weiden i.d.OPf.", "09363", "https://www.weiden.de/kultur/ehrenamt/ehrenamtskarte"),
+    listOf(
+        "Landkreis",
+        "Weilheim-Schongau",
+        "09190",
+        "https://www.weilheim-schongau.de/buergerservice/ehrenamtskarte/"
+    ),
+    listOf("Landkreis", "Wunsiedel i.Fichtelgebirge", "09479", "http://www.landkreis-wunsiedel.de/ehrenamtskarte")
+)
+
+fun setupDatabase() {
     SchemaUtils.create(
         Regions
     )
 
-    executeScript("sql/create_regions.sql")
+    transaction {
+        val projects = ProjectEntity.all()
+        val dbRegions = RegionEntity.all()
+
+        // Create a mock region for each project to save the stores to
+        projects.forEach { project ->
+            // Project ids and therefore the mock regions should never change
+            // Therefore we only need to create the mock region if not already existing
+            if (dbRegions.none { it.regionIdentifier == project.regionIdentifier() }) {
+                // TODO #538: Perhaps no mock region will be needed anymore
+                RegionEntity.new {
+                    projectId = project.id
+                    name = project.project
```

Comment:
optional:
```suggestion
                    name = "DUMMY FOR " + project.project
```

Reply:
As I have to use the name to query the region I'd keep it as just the project id without the dummy.